src/AppBundle/Resources/translations/report-money-transaction.en.yml

```diff
@@ -273,7 +291,7 @@ form:
         label: Investment account
         moreInformations: For each new investment, please enter the date, a description and the value
       purchase-over-1000:
-        label: Purchases over £1,000
+        label: Other purchases over £1,000
```

Comment:
Should this be singular (purchase over £1000)? As per the "Major purchase over £1000" above ?

Reply:
yes, Uchenna suggested all those to be singular. Fixing.